conda/recipes/cudf/meta.yaml

```
@@ -57,7 +62,14 @@ requirements:
     - pyarrow =11
     - libcudf ={{ version }}
     - rmm ={{ minor_version }}
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
```

Is `cuda-version` sufficient for constraining `cudatoolkit` ([ref]( https://github.com/conda-forge/cuda-version-feedstock/blob/3ce1bae701079226f982b173667edc5d0a64c01f/recipe/meta.yaml#L17 ))?

```suggestion
    - cudatoolkit
```